products/emberjs.md

```diff
@@ -38,10 +38,18 @@ auto:
 # - eoas(x) = releaseDate(x+1)
 # - eol(x) = releaseDate(x+1)
 releases:
+-   releaseCycle: "5.12"
+    releaseDate: 2024-10-04
+    lts: 2024-11-15
+    eoas: 2025-08-01
```

Comment:
https://www.timeanddate.com/date/dateadded.html?d1=15&m1=11&y1=2024&type=add&ay=&am=&aw=36&ad=&rec=
```suggestion
    eoas: 2025-07-25
```
WDYT?

Reply:
yeah, that looks right, I was using gpt to the calc, apparently, it is very unreliable